.sample-env.md

```diff
@@ -1 +1,30 @@
-COVERALLS_REPO_TOKEN=LBqOgmrFuiVAaGMil54sD40Axr58PJ7kz
```

Comment:
nice work, build is failing tho please check it. also you can fix db issue as well